sentry/src/main/java/io/sentry/Scopes.java

```diff
@@ -717,8 +736,7 @@ public void flush(long timeoutMillis) {
     if (!isEnabled()) {
       getOptions().getLogger().log(SentryLevel.WARNING, "Disabled Scopes cloned.");
     }
-    // TODO [HSM] should this fork isolation scope as well?
-    return new HubScopesWrapper(forkedCurrentScope("scopes clone"));
+    return new HubScopesWrapper(forkedScopes("scopes clone"));
```

Comment:
Now forking both current and isolation scope since that's closer to what cloning a hub did.